### PR TITLE
Improvement/awt 43 add more locators

### PR DESCRIPTION
**Pull request by rrchcr8**
ya esta Marines para que fixees tu branch

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java [hunk 1]
```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
         return false;
     }
 
+    /**
+     * click dropdown story button..
+     *
+     * @param text name of the history
+     */
+    public void clickstoryDropdownButton(final String text) {
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/Story.java]: Please move it to Panel page and correct the name to : expandStory()

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java [hunk 2]
```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
...
+                .concat("']/ancestor::header/child::button");
+        action.click(By.xpath(xpathExpression));
+    }
+
+    /**
+     * click delete button.
+     */
+    public void clickDeleteButton() {
+        this.action.click(this.deleteStoryButton);
+
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/Story.java:157]: please remove unnecessary empty lines

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java [hunk 3]
```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
...
+    }
+
+    /**
+     * click delete button.
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/Story.java]: please use single comment :

/** Click delete button. **/

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java [hunk 4]
```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
...
+
+    }
+
+    /**
+     * select an especific checkbox.
+     *
+     * @param text is the name of a speceif story .
+     */
+    public void clickStoryCheckboxButton(final String text) {
+        for (final WebElement element : this.storyCheckBoxes) {
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/Story.java]: please add same approach than expandStory using xpath

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java [hunk 5]
```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
...
+            element.click();
+            clickDeleteButtonOfToast();
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/Story.java]: don't forget refactor it creating DeleteDialog page, check if this dialog is used in workspace and project, here or in new branch

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java [hunk 6]
```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
...
+            clickConfirmDeleteButton();
+        }
+
+    }
+
+    /**
+     * this method clicks the delete button.
+     */
+    public void clickDeleteButtonOfToast() {
+        this.action.click(this.deleteStoryButtonOfToast);
```
Comment (88mary256) [inline src/main/java/org/fundacionjala/pivotal/pages/Story.java]: Please create toast page and move this method there, in this or a new branch.

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java [hunk 7]
```diff
@@ -63,4 +61,44 @@ public void verifytheStoryIsCreated() {
...
         story.createStory(arg1);
     }
+
+    /**
+     * This method clicks the expand button for a specific story.
+     *
+     * @param storyKeyName the name of the story.
+     */
+    @When("selects the dropdown button of the story {string}")
+    public void selectsTheDropdownButtonOfTheStory(final String storyKeyName) {
```
Comment (88mary256) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java]: please rename it to expandstory

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java [hunk 8]
```diff
@@ -63,4 +61,44 @@ public void verifytheStoryIsCreated() {
...
+        story.clickConfirmDeleteButton();
+    }
+
+    /**
+     * Not a reliable method.
+     *
+     * @param arg0 not really sure (it was in progress).
+     */
+    @When("deletes selecting the checkboxof {string}")
+    public void deletesSelectingTheCheckboxof(final String arg0) {
```
Comment (88mary256) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java]: please use a better name param than arg0

File: src/test/resources/features/Steps_part2.feature [hunk 9]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (88mary256) [inline src/test/resources/features/Steps_part2.feature]: Rename this file for a better name than steps: it is not a steps class, it is a feature file with scenarios.
Maybe a name that describe background.

**Review by 88mary256 — CHANGES_REQUESTED** [review]

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java (hunk 1 above)
Comment (rrchcr8) [reply src/main/java/org/fundacionjala/pivotal/pages/Story.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java (hunk 2 above)
Comment (rrchcr8) [reply src/main/java/org/fundacionjala/pivotal/pages/Story.java:157]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java (hunk 3 above)
Comment (rrchcr8) [reply src/main/java/org/fundacionjala/pivotal/pages/Story.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java (hunk 4 above)
Comment (rrchcr8) [reply src/main/java/org/fundacionjala/pivotal/pages/Story.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java (hunk 5 above)
Comment (rrchcr8) [reply src/main/java/org/fundacionjala/pivotal/pages/Story.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/main/java/org/fundacionjala/pivotal/pages/Story.java (hunk 6 above)
Comment (rrchcr8) [reply src/main/java/org/fundacionjala/pivotal/pages/Story.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java (hunk 7 above)
Comment (rrchcr8) [reply src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java (hunk 8 above)
Comment (rrchcr8) [reply src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

File: src/test/resources/features/Steps_part2.feature (hunk 9 above)
Comment (rrchcr8) [reply src/test/resources/features/Steps_part2.feature]: fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0

**Review by 88mary256 — APPROVED** [review]

**Pull request merged**